DemoApp/routes/report.router.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const router = express.Router();
+const multer = require("multer");
+const uploadFile = require("../middlewares/upload");
+const {upload} = require("../controller/report.controller");
+const {userAuth}=require('../controller/user.controller')
+const FileType = require('file-type');
+const fs = require('fs')
+const { promisify } = require('util')
+
+const unlinkAsync = promisify(fs.unlink)
+
+const {
+    successResponse,
+    errorResponse}
+    =require('../helpers/helpers')
+
+    const {
+        PLEASE_SELECT_FILE,
+        UPLOADED_SUCCESSFULLY,
+        FILE_SIZE,COUDNT_UPLOAD}
+        =require('../helpers/messages')
+      
+        const whitelist = [
+            'application/pdf',
+            'application/msword',
+            'application/vnd.openxmlformats-officedocument.wordprocessingml.document'
+          ]
+          
+// router.post('/upload',uploadFile,upload);
+router.post('/upload',uploadFile,async (req, res,next) => {
+    try {
+        console.log(req.file)
+        const meta = await FileType.fromFile(req.file.path)
```

Comment:
You can use something like this to get all files ->
req.files.forEach(file => { console.log(file.path) });